SUPPORT.md

```diff
@@ -0,0 +1,46 @@
+# Support
+
+This article explains where to get help with Shipwright projects.
+Please read through the following guidelines.
+
+> 👉 **Note**: before participating in our community, please read our
+> [code of conduct](CODE_OF_CONDUCT.md).
+> By interacting with this repository, organization, or community you agree to
+> abide by its terms.
+
+## Slack
+
+Participate in near real-time conversations with other Shipwright community members. Ask questions, get answers, build relationships.
+
+[Join the Channel](https://kubernetes.slack.com/archives/C019ZRGUEJC)
```

Comment:
[Join the Kubernetes Slack workspace if you haven't already](slack.kubernetes.io), then [join the #shipwright channel](https://kubernetes.slack.com/archives/C019ZRGUEJC).